README.md

```diff
@@ -50,6 +50,223 @@ This app has the following default slide separators:
+      text: <text-for-title1>
+    - title: <about-us-title2>
+      text: <text-for-title2>
+    - title: <about-us-title3>
+      text: <text-for-title3>
+---
+```
+
+These are the supported metadata required for the presentation template.
+
+| Default Metadata | Description                                                                                                                                                |
+|------------------|------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `slide`          | Default slide for presentation. Most used slide template can be given in the default metadata.                                                             |
+| `presenter`      | Name of the presenter.                                                                                                                                     |
+| `logo`           | Logo appears on the top right corner of the slide. The logo path should be relative to the markdown file.                                                  |
+| `color`          | Font color of the slide title.                                                                                                                             |
+| `footer`         | Content to be shown in the footer of the slide.                                                                                                            |
+| `aboutUs`        | Column wise content to be shown in the about-us slide. Use this metadata only if about-us slide is used for the presentation. More [here](#slide-about-us) |
+| `aboutUs:title`  | Title for each column in about-us slide.                                                                                                                   |
+| `aboutUs:text`   | Text under each title for column in about-us slide.                                                                                                        |
+
+Following the front matter, create slides as described in [Creating Presentation](#creating-presentation)
+
+Besides default metadata, you can also provide inline metadata for each slide. This will be useful when you want to
+override the default metadata for a specific slide and this will be applied to that slide only. The inline metadata can
+be defined by adding metadata directly on the slide title as below:
+
+```markdown
+# Title of the slide ::metadata_key_1:metadata_value_1 ::metadata_key_2:metadata_value_2
+```
+
+Example:
+```markdown
+# Title of slide ::slide:title-content-image ::logo:logo.png
+```
+
+For this slide, the `slide` template will be set to `title-content-image`, and the `logo` will be set to `logo.png`.
+
+### Slide Templates
+
+The slide template should be provided in the default metadata. Provide slide template that will be mostly used in the
+default metadata, so you don't need to add the slide metadata in every slide.
```

Comment:
This is confusing, I want to  have different templates for different slides. But the template that is mostly used should be set in the default. Is that what you want to say?